### DISCUSSION TITLE
Is this mania editor description deliberate or hindsight?

### DISCUSSION THREAD

**Discussion by aryoadhi** [ppy/osu#23426] [Q&A] [answered]
![osu_2023-05-08_07-15-03](https://user-images.githubusercontent.com/5029654/236710910-4713bb78-da83-492f-9e64-85df530b92e8.jpg)

The special (N+1) style mentions **5K (4+1)** and **8key (7+1)**.

I've never seen any odd-count styles (5K, 7K, 9K) that incorporates the S button to be moved to left and right, so I think it might be a hindsight. Most common ones I seen are 6K (5+1) and 8K (7+1), due to 5K and 7K already have special key at the center and mostly associated with spacebar however.

It would be great if someone show me 4+1 and 6+1 style in other video games as the reference if it's really a deliberate wording.

**Comment by peppy** [comment, marked as answer]
It's incorrect wording, should be 6 and 8, not 5 and 8.

**Reply by aryoadhi** [reply]
Thanks for the clarification. I was thinking that one can stretch 4+1 with IIDX style and excluding the top 3 buttons that makes it 7+1.
